Tests/Functional/Domain/Repository/Fixtures/Product/Tea.csv

```diff
@@ -1,8 +1,8 @@
 "tx_tea_domain_model_product_tea"
-,"uid","pid","title","description","image"
-,"1","1","Earl Grey","Fresh and hot.",0
-,"2","1","Assam","Dark and strong.",0
-,"3","1","Gunpowder","Bitter and very green.",1
+,"uid","pid","title","description","image","owner"
```

Comment:
Nothing for this PR, but a general question: Should we discuss the CSV Format? I know this is the only official way provided by the testing framework. But our company has no good relation to CSV, we prefer PHP.

Maybe something to discuss within our regular team call, what do you think?

Reply:
Yes, let's move this discussion to our call.

Reply:
Done: https://notes.typo3.org/JKT4Cr6tSZehjq0Nu8o65g?both#discussion-keep-csv-for-database-fixtures